plugins/main/bridge/bridge.go

```diff
@@ -127,6 +142,59 @@ func loadNetConf(bytes []byte, envArgs string) (*NetConf, string, error) {
 	return n, n.CNIVersion, nil
 }
 
+// This method is copied from https://github.com/k8snetworkplumbingwg/ovs-cni/blob/v0.27.2/pkg/plugin/plugin.go
+func collectVlanTrunk(vlanTrunk []*VlanTrunk) ([]int, error) {
+	if vlanTrunk == nil {
+		return nil, nil
+	}
+
+	vlanMap := make(map[int]bool)
```

Comment:
might as well use a `map[int]struct{}` which I think is more memory efficient. IIUC, you're using a set (you only care about the keys, and never put the value `false`  in the map).

I also think it is more idiomatic, since the type clearly shows there's nothing in the value to be looked at.

Reply:
ok got it.